pkgs/tools/misc/pistol/default.nix

```diff
@@ -6,16 +6,16 @@
 
 buildGoModule rec {
   pname = "pistol";
-  version = "0.0.5";
+  version = "0.1";
 
   src = fetchFromGitHub {
     owner = "doronbehar";
     repo = pname;
     rev = "v${version}";
-    sha256 = "1zxvpmy6c26dhg5wpk5cgaqipw5372crkhm1vaghm8jkiw7sjwvw";
+    sha256 = "0za0l2dhgwfwgyjclajbm97by5a8w7q1mcq9czghxag19nfpgwf7";
   };
 
-  modSha256 = "13yxcfd29ziprjsjl2ji7w5i2506hwwl3y0ycaphj2wlcd75rdxs";
+  modSha256 = "0r062nka72ah2nb2gf8dfrrj4sxadkykcqjzkp4c9vwk93mhw41k";
 
```

Comment:
would you mind adding?
```
  buildFlagsArray = [ "-ldflags=-s -w -X main.VERSION=${version}" ];
```

Reply:
It didn't work for me before from some reason, no idea why. Now it works.